webapps/src/main/webapp/WEB-INF/conf/task-addon/spaces-templates-configuration.xml

```diff
@@ -16,95 +16,105 @@
-            <init-params>
-                <object-param>
-                    <name>template</name>
-                    <description>Space Template</description>
-                    <object type="org.exoplatform.social.core.space.SpaceTemplate">
-                        <field name="name"><string>project</string></field>
-                        <field name="applications">
-                            <collection type="java.util.ArrayList">
-                                <value>
-                                    <object type="org.exoplatform.social.core.space.SpaceApplication">
-                                        <field name="portletApp">
-                                            <string>TasksManagement</string>
-                                        </field>
-                                        <field name="portletName">
-                                            <string>TasksManagement</string>
-                                        </field>
-                                        <field name="appTitle">
-                                            <string>Tasks</string>
-                                        </field>
-                                        <field name="removable">
-                                            <boolean>true</boolean>
-                                        </field>
-                                        <field name="order">
-                                            <int>500</int>
-                                        </field>
-                                        <field name="uri">
-                                            <string>tasks</string>
-                                        </field>
-                                    </object>
-                                </value>
-                            </collection>
-                        </field>
-                    </object>
-                </object-param>
-            </init-params>
-        </component-plugin>
-        <!-- end Project space template -->
-    </external-component-plugins>
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd http://www.exoplatform.org/xml/ns/kernel_1_2.xsd"
+  xmlns="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd">
+  <external-component-plugins>
+    <target-component>org.exoplatform.social.core.space.spi.SpaceTemplateService</target-component>
+    <!-- Community space template -->
+    <component-plugin>
+      <name>Space Template Extension</name>
+      <set-method>extendSpaceTemplatePlugin</set-method>
+      <type>org.exoplatform.social.core.space.SpaceTemplateConfigPlugin</type>
+      <init-params>
+        <object-param>
+          <name>template</name>
+          <description>Space Template</description>
+          <object type="org.exoplatform.social.core.space.SpaceTemplate">
+            <field name="name">
+              <string>community</string>
+            </field>
```

Comment:
Thos Space templates aren't needed in Meeds context, can you please move it to eXo addons context.

```suggestion
```

Reply:
@boubaker we already have the configuration of all space templates in social, I have seen that in meeds that we [override ](https://github.com/Meeds-io/meeds/blob/develop/webapps/plf-meeds-extension/src/main/webapp/WEB-INF/conf/social-extension/social/spaces-templates-configuration.xml) the original social configuration with crowdFunding, do you mean to keep the main configuration of all space templates in social and move the others (task, notes, wallet) to platform-private-distribution ? can you explain that ?

Reply:
should we move the current social configuration also @boubaker ?

Reply:
Move all of 4 useless templates, yes please.

Reply:
@boubaker seen with the guild, since this is a cleaning not related closely to this MIP, can we note this action to be done later for 6.6 ?

Reply:
ok, see answer here https://github.com/Meeds-io/notes/pull/833#issuecomment-1825525891